### ISSUE TITLE
Please add model cumulative cases

### ISSUE THREAD

**Issue by coderb** [neherlab/covid19_scenarios#517]
Please add data for the model cumulative cases.

This would be useful when trying to manually fit parameters to observed data especially around interventions.

**Comment by noleti** [comment]
Is this the same as https://github.com/neherlab/covid19_scenarios/issues/504 ?

**Comment by coderb** [comment]
no, it's a request for a new data series which is not currently displayed

**Comment by noleti** [comment]
Can you be more specific? You meant the sum of people who were infected, in hospital, in ICU, or dead? I.e. total population-recovered?

**Comment by coderb** [comment]
the cumulative number of people that have been infected over time, ie the model version of the data series "cumulative cases (data)".

**Comment by ccpf** [comment]
Isn't that: cumulative deaths + icu + severely ill + infectious + recovered? At least this is how I have been interpreting the model output.

**Comment by noleti** [comment]
AFAIK "Cumulative cases (data)" is directly taken from our `data/case-counts/*/*.tsv` files (which are, in turn, taken from third party APIs). So you could directly take the values from our repo. Does that help?

**Comment by ccpf** [comment]
@noleti: true! Thanks for pointing that out. So then coderb is right in that we cannot know the (modelled) cumulative cases? Regarding your answer to him/her, I think the question was about the modelled cumulative cases and not the actual data. Cheers.

**Comment by coderb** [comment]
@noleti i think you misunderstand what i am asking for.  simply to plot (in the graph section) what the model gives for the cumulative number of cases.  i believe the formula would simply be N - S where N is the total population and S is the number of susceptible (assuming there is no change in total population over time).

**Comment by noleti** [comment]
Your N-S is the number I suggested in https://github.com/neherlab/covid19_scenarios/issues/517#issuecomment-612770785. I'm not sure how this will help with manual fitting, as usually there is a big problem with undercounting of infected (but not severe) cases. Fitting is most reliably done based on deaths. 

In any case, I think it is now clear what you are asking for, I will update your initial question to describe it more clearly from my perspective.

**Comment by coderb** [comment]
thank you

**Comment by nnoll** [comment]
Hey @coderb. My worry about including this is as @noleti states, it's probably not wise to fit our cumulative infectious categories directly to the cumulative case counts as there's an unknown underreporting fraction for each scenario. Furthermore, I worry about our UX here: our results plot is already quite dense with information that I fear adding another plot will compound our problem there.

One possibility is we could break up the plot into "current" and "cumulative" reports. This would allow us to plot the data and maybe is better from an interpretability standpoint. @rneher and @ivan-aksamentov thoughts?

**Comment by coderb** [comment]
@nnoll regarding the information overload, perhaps just defaulting the data series to unselected would work.

i understand you should not directly fit the cumulative cases, however, i'm interested in the overall shape.  additionally, that unknown underreporting fraction seems to be a pretty important factor in the model's prediction of the future fatality curve.

i've been working on improving the age specific parameter matrix to make more sense for my particular location which i think i have done, however without the model cum cases it's hard to get a good sense for how the other parameters effect the model.  thanks.

**Comment by coderb** [comment]
@nnoll the actual number of deaths is not as reliable as you would think:
[New York to start reporting 'probable' coronavirus deaths to CDC](https://www.nbcnews.com/news/us-news/new-york-start-reporting-probable-coronavirus-deaths-cdc-n1184821)

**Comment by ccpf** [comment]
@coderb 
yes, an in italy also according to this large scale analysis of "excess deaths" in over 1000 communities: http://www.deplazio.net/images/stories/SISMG/SISMG_COVID19.pdf
here is some summary/analysis in English: https://towardsdatascience.com/covid-19-excess-mortality-figures-in-italy-d9640f411691
suggesting that official deaths should be multiplied by a factor of about 2.5

**Comment by shijurodhaz** [comment]
<img width="597" alt="Screen Shot 2020-04-21 at 4 11 00 AM" src="https://user-images.githubusercontent.com/22701131/79842055-54377b80-8386-11ea-9250-3bf24b87eb83.png">
I submitted a pull request for these changes. I basically did population - current susceptible as mentioned in the issue title. Let me know if anything needs to be modified. Thanks!

**Comment by amindelpazir** [comment]
#695
